astroplan/plots/plots.py

```diff
@@ -0,0 +1,216 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+import random
+
+
+def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
+                 dark_plot=False):
+    """
+    TODO:
+        1) Timezones?
+        2) Limit airmass <= 3.
+        3) Plot lines instead of points.
+        4) Legend should not be transparent.
+        5) dark_plot option
+
+    Makes an airmass vs. time plot.
+
+    If an ax object is passed in, plots an additional airmass plot on top.
+    Otherwise, creates a new ax object with an airmass plot.
+
+    When a `Time` object with a single instance in time is used (e.g., if
+    len(time) = 1), a new `Time` object is created internally.  This new object
```

Comment:
One clarification here: it is *not* correct that a scalar time (which is what you mean here) is the same as one with ``len(time) == 1``.  For example:
```python
>>> Time(['2010-1-1']).isscalar
False
>>> len(Time(['2010-1-1']))
1
>>> Time('2010-1-1').isscalar
True
>>> len(Time('2010-1-1'))
TypeError: Scalar Time object has no len()
```
So in the docs you want to be very clear about this: it's not ``a Time object with a single instance`` but rather ``a Time object with a scalar value (e.g. if `time.isscalar` is true) ``